mantid-developer-linux.yml

```diff
@@ -0,0 +1,68 @@
+name: mantid
+
+channels:
+  - conda-forge
+  - DLR-SC # Required for OpenCascade
+
+dependencies:
+  - python=3.8.*
+  - cmake>=3.20.*
+  - texlive-core>=20180414
+  - doxygen>=1.9.*
+  - tbb=2020.2.*
+  - tbb-devel=2020.2.*
+  - boost=1.75.* # Also pulls in boost-cpp
+  - poco=1.10.*
+  - nexus=4.4.*
+  - hdf5=1.10.*
+  - jemalloc>=5.2.1
+  - gsl=2.6 # Keep gsl a specific version to reduce changes in our fitting
+  - jsoncpp>=1.9.4,<2
+  - ninja>=1.10.2
+  - graphviz>=2.47.0
+  - librdkafka>=1.6.0
+  - eigen=3.3.*
+  - opencascade>=7.4.0
+  - zlib>=1.2.11
+  - muparser>=2.3.2
+  - qt>=5.12.9,<6
+  # Not Windows, OpenGL implementation:
+  - mesalib>=18.0.0
+  - mesa-libgl-devel-cos7-x86_64>=18.3.4
+  - mesa-libgl-cos7-x86_64>=18.3.4
+  # Needed for test suite on Linux
+  - xorg-libxcomposite>=0.4.5
+  - xorg-libxcursor>=1.2.0
+  - xorg-libxdamage>=1.1.5
+  - xorg-libxi>=1.7.10
+  - xorg-libxtst>=1.2.3
+  - xorg-libxscrnsaver>=1.2.3
+  - pciutils-libs-cos7-x86_64>=3.5.1
+  # Linux only
+  - gxx_linux-64==9.3.*
+  - libglu>=9.0.0
+  # Python Packages:
+  - setuptools=48.0.0 # Pinned purposefully due to incompatibility with later versions
+  - qtwebkit>=5.212,<6
```

Comment:
Do you know what still requires the Qt WebKit? I thought that was only for Qt4.

Reply:
I can confirm that if I remove `qtwebkit` and create and environment locally then I able to compile mantid.

Reply:
Removed